dashboard/lib/build_dashboard_page.dart

```diff
@@ -223,22 +223,22 @@ class BuildDashboardPageState extends State<BuildDashboardPage> {
           branch = selectedBranch;
           _updateNavigation(context, _buildState);
         },
-        items:
-            // FOR REVIEW: to ensure a valid transition no matter what invalid combinations of repo + branch user inputed,
-            // we preserve that invalid combination as a single entry, along with the other valid combinations
-            [
+        items: [
+          // To ensure a valid transition no matter what invalid combinations of repo + branch user inputed,
+          // we preserve that invalid combination as a single entry, along with the other valid combinations
+          // corresponding test is "ensure smooth transition between invalid state"
           DropdownMenuItem<String>(
```

Comment:
nit: we don't leave comments like this. Well written tests should make it obvious the functionality they're testing

Reply:
Fwiw, the test.looks well documented to me